core/templates/base-components/i18n-footer.component.html

```diff
@@ -0,0 +1,13 @@
+<ul>
+  <li>
+    <select [(ngModel)]="currentLanguageCode"
```

Comment:
Optional: Maybe change the component name to i18n-language-selector or site-language-selector?

Reply:
done